### PR TITLE
virt plugin: fix skipping inactive domains

### PR DESCRIPTION
**Pull request by ipavlikevich**
This patch fixes collecting disk/iface metrics from paused domains and prevents failures in virDomainBlockStats()/virDomainInterfaceStats() with timeout error:
```
libvirtd: Timed out during operation: cannot acquire state change lock
collectd: libvirt: QEMU Driver error : Timed out during operation: cannot acquire state change lock
```

Signed-off-by: Igor Pavlikevich <ipavlikevich@gmail.com>

### CODE REVIEW

**Comment by rubenk** [comment]
Thank you. This looks like the correct fix, but I'll hold off merging for a bit since large parts of the virt plugin are being rewritten as we speak. See #2037 for the discussion.

**Comment by rubenk** [comment]
Thanks. Can you please rebase?

**Comment by ipavlikevich** [comment]
Rebased

File: src/virt.c [hunk 1]
```diff
@@ -1143,6 +1140,18 @@ static int refresh_lists(struct lv_read_instance *inst) {
         goto cont;
       }
 
+      status = virDomainGetInfo(dom, &info);
+      if (status != 0) {
+        ERROR(PLUGIN_NAME " plugin: virDomainGetInfo failed with status %i.",
```
Comment (rubenk) [inline src/virt.c]: Should this really be logged as an error? I can imagine it happens quite often that a domain is stopped or disappears between the lookup above and this call. Perhaps a warning is more appropriate?

Comment (ipavlikevich) [inline src/virt.c]: There are two calls directly above, virDomainLookupByID() and virDomainGetName(), so this exception must be rather rare situation.

Comment (rubenk) [inline src/virt.c]: Ok, we'll see how it goes. If it shows up too much we can always downgrade it.

**Pull request merged**